ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureCloudConfig.java

```diff
@@ -272,6 +404,7 @@ public AzureCloudConfig(VerifiableProperties verifiableProperties) {
     cosmosKey = verifiableProperties.getString(COSMOS_KEY, "");
     cosmosKeySecretName = verifiableProperties.getString(COSMOS_KEY_SECRET_NAME, "");
     cosmosVaultUrl = verifiableProperties.getString(COSMOS_VAULT_URL, "");
+    getAzureStorageAccountInfo = verifiableProperties.getString(AZURE_STORAGE_ACCOUNT_INFO, "");
```

Comment:
As mentioned earlier, we should parse the config here.

Reply:
Done